server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -282,6 +286,8 @@ private void handleProvideEvidence(JSONObject evidence, Player player) {
 	  evidence.put(Constants.EVENT_TYPE, EventType.EVIDENCE_PROVIDED_NOTIFICATION);
 	  evidence.put(Constants.AUTHOR, player.getTag());
 	  suggester.sendEvent(evidence);
+	  handleSimpleEvent(suggester, EventType.PREVENT_SUGGEST);
+	  handleSimpleEvent(suggester, EventType.ALLOW_TURN_END);
```

Comment:
Whats the need for these events? Can't the frontend decide when these things are allowed without an event front the backend?

ie. disable the suggestion button after its been pressed, and enable it when its your turn
ie. only allow end turn after movement has been made